ci/scripts/check-bazel-tags.sh

```diff
@@ -6,14 +6,37 @@
 
 set -e
 
-untagged=$(./bazelisk.sh query "rdeps(//..., //hw:verilator) except attr(tags, '[\\[ ](verilator|manual)[,\\]]', //...)")
+untagged=$(./bazelisk.sh query "rdeps(//..., //hw:verilator) except attr(tags, '[\\[ ](verilator|manual)[,\\]]', //...)" --output=label_kind)
 if [[ ${untagged} ]]; then # Check that all targets that depend on verilator are tagged
-  echo "Target(s): ${untagged} depend(s) on //hw:verilator, please tag it with verilator or manual";
+  echo "Error:";
+  echo "${untagged}"|sed 's/^/    /';
+  echo "Target(s) above depend(s) on //hw:verilator, please tag it with verilator or manual.";
   exit 1
 fi
 
-untagged=$(./bazelisk.sh query "rdeps(//..., kind('bitstream_splice', //...)) except attr(tags, '[\\[ ](cw310_rom|cw310_test_rom|vivado|manual)[,\\]]', //...)")
+untagged=$(./bazelisk.sh query "rdeps(//..., kind('bitstream_splice', //...)) except attr(tags, '[\\[ ](cw310_rom|cw310_test_rom|vivado|manual)[,\\]]', //...)" --output=label_kind)
 if [[ ${untagged} ]]; then # Check that all targets that depend on vivado are tagged
-  echo "Target(s): ${untagged} depend(s) on a bitstream_splice but isn't tagged with vivado or manual";
+  echo "Error:";
+  echo "${untagged}"|sed 's/^/    /';
+  echo "Target(s) above depend(s) on a bitstream_splice and isn't available in a cache."
+  echo "Please tag it with vivado or manual.";
+  exit 1
+fi
+
+mistagged=$(./bazelisk.sh query "rdeps(attr(tags, '[\\[ ]cw310_test_rom[,\\]]',//...), kind('bitstream_splice', //...) except deps(//hw/bitstream:test_rom))" --output=label_kind)
```

Comment:
Nit: Could you try to define a function that defines the tag-matching patterns? I think it would improve readability.

This is getting complicated enough that I would support rewriting it in anything other than bash... but maybe another week :)